### PR TITLE
Prepare for version 11.8.0

### PR DESCRIPTION
**Pull request by j-rivero**

### CODE REVIEW

**Review by chapulina — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+
+1. Enable hardware PCF for spotlight shadows
+    * [Pull request #3036](https://github.com/osrf/gazebo/pull/3036)
+
+1. Apply Force/Torque for nested models
+    * [Pull request #3039](https://github.com/osrf/gazebo/pull/3039)
+    * A contribution by Cameron Miller
+
+1. Make links within nested models modifiable from GUI Client
+    * [Pull request #3039](https://github.com/osrf/gazebo/pull/3039)
```
Comment (scpeters) [inline Changelog.md]: #3039 -> #3031

also add #3059, which was a follow-up to #3031

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+    * A contribution by Sonia Jin (Amazon)
+
+1. Make events threadsafe
+    * [Pull request #3042](https://github.com/osrf/gazebo/pull/3042)
+    * A contribution by Sonia Jin (Amazon)
+
+1. Add relocatable setup.bash script
+    * [Pull request #3061](https://github.com/osrf/gazebo/pull/3061)
```
Comment (scpeters) [inline Changelog.md:28]: optional: resolved issue #3056

**Review by scpeters — COMMENTED** [review]
a couple adjustments to the changelog are needed:

* https://github.com/osrf/gazebo/compare/gazebo11_11.7.0...gazebo11

I'll make these changes later today if they haven't already been done

File: Changelog.md (hunk 1 above)
Comment (scpeters) [reply Changelog.md]: fixed in 7b391b1

File: Changelog.md (hunk 2 above)
Comment (scpeters) [reply Changelog.md:28]: fixed in 7b391b1

**Pull request merged**